assets/components/subscriptionsByCountryGroup/components/digitalSection.jsx

```diff
@@ -30,7 +30,7 @@ type PropTypes = {|
 
 function DigitalSection(props: PropTypes) {
   return (
-    <ThreeSubscriptions heading="Digital Subscriptions">
+    <ThreeSubscriptions heading="">
```

Comment:
Leaving the heading blank here because otherwise it clashes with the subscribe text in the header when there is no featured product